src/main/java/ch/ivyteam/ivy/maven/compile/AbstractProjectCompileMojo.java

```diff
@@ -48,6 +48,10 @@ public abstract class AbstractProjectCompileMojo extends AbstractEngineInstanceM
   @Parameter(property = "ivy.compiler.warnings", defaultValue = "true")
   boolean compilerWarnings;
 
+  /** Defines the source output directory for annotation-processors. */
+  @Parameter(property = "ivy.compiler.apt.source.output")
+  private String generatedSourcesDirectory;
```

Comment:
align the parameter name with the native compiler-plugin: 
https://www.baeldung.com/java-annotation-processing-builder#3-using-maven

Reply:
requires: https://github.com/axonivy/core/pull/9710